README.rst

```diff
@@ -1,130 +1,165 @@
-You might also want to visit `our website <http://coala-analyzer.org/>`_.
+.. image:: https://cloud.githubusercontent.com/assets/7521600/15992701/ef245fd4-30ef-11e6-992d-275c5ca7c3a0.jpg
+
+coala: Language Independent Code Analysis
+-------------
+**coala provides a common command-line interface for linting and fixing all
+your code, regardless of the programming languages you use.**
+
+With coala, users can create rules and standards to be followed in the source
```

Comment:
maybe we should link them to how to create a coafile so they can reed more there if interested how they create rules and standards? have it: ``can create `rules and standards <link>`__ to be..``